projects/novo-examples/src/updates/v10.md

```diff
@@ -0,0 +1,48 @@
+---
+section: updates
+page: v10-announce
+title: v10
+order: 5
+tag: new
+---
+
+🎉 April 2024 (version 10)
+===========================
+
+**Announcement**: Novo Elements is now on Angular 17!
+
+This brings Novo Elements up to the latest versions of both Angular and Typescript. We have also removed the long-since deprecated Table component and Dragula add-on. 
+
+## Upgrading to v10
+- You may need to update your node version as Angular 17 requires at least `^18.13.0` or `^20.9.0`.
```

Comment:
Ensure your node version is compatible with Angular 17 which requires `^18.13.0` or `^20.9.0`.